src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/PaymentAccountServiceImplTest.java

```diff
@@ -100,37 +100,38 @@ public void retrievePaymentAccountsByPbaEmailWhenConfigTrue() {
 
         verify(organisationMock, times(1)).setPaymentAccounts(any());
         verify(organisationMock, times(1)).setUsers(any());
-        verify(organisationMock, times(1)).setPaymentAccounts(any());
         verify(professionalUserRepositoryMock, times(1)).findByEmailAddress("some-email");
     }
 
+    @Test(expected = Exception.class)
+    public void testThrowsExceptionWhenEmailInvalid() {
+        when(sut.findPaymentAccountsByEmail("some-email")).thenReturn(organisation);
```

Comment:
actual method is not called, just used when

Reply:
updated now